### PR TITLE
Signin script updated

### PR DESCRIPTION
**Pull request by avishekjana**
#67

### CODE REVIEW

File: making-api-requests/user-signin/README.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+# Script
+Hey there! In this lesson you will learn to create a **user signin** form in React. We'll be using `async/await` to make an API call to authenticate the user, and **React Router** to redirect the user after successful sign-in.
+
+So let's get started.
+
+First, we will start with adding the route for the signin page. For that, I'll create a new  `signin` folder inside the `pages` directory.
```
Comment (reenas) [inline making-api-requests/user-signin/README.md:6]: @avishekjana Where is the `pages` directory in our `smarter-tasks` folder? Are we creating bot `pages` and `signin` in this level fresh? Please double-check for continuity from Level 5. I did not notice any `pages` folder there.

File: making-api-requests/user-signin/README.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      <h1>Sign in</h1>
+    </div>
+  );
+}
+
+// Dialogue 3: And finally, we've to export the component
+export default Signin;
+```
+
+Let's add this new `Signin` component in our `App` component.
```
Comment (reenas) [inline making-api-requests/user-signin/README.md:27]:
```suggestion
Let's add this new `Signin` component in our `App` component, i.e., _App.tsx_ file.
```

File: making-api-requests/user-signin/README.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+          <Route exact path="/signin" element={Signin} />
+        </Switch>
+      </AuthProvider>
+    </Router>
+  );
+};
+export default App;
+```
+Now, let's go back to the browser to check if Signin page is coming or not.
+> Action: Open browser and show the signin page.
```
Comment (reenas) [inline making-api-requests/user-signin/README.md:53]: @avishekjana Mention the URL that students should open in the browser.

File: making-api-requests/user-signin/README.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+Here, when the form is submitted, the `handleSubmit` function is called, which makes a POST request to the `/api/signin` endpoint with the email and password from the form. If the request is successful, the function logs a message to the console, and redirects the user to the `/dashboard` path using the `history` object from `react-router-dom`. If the request fails, an error message is logged to the console.
+
+Next, we've to add this form in our Signin page, and that's pretty much straight-forward.
+```tsx
+import React, { useState } from 'react';
+// Dialogue 1: Just import the file
+import SigninForm from "./SigninForm"
+
+const Signin: React.FC<> = () => {
+  // Dialogue 2: And use it after the h2 tag
```
Comment (reenas) [inline making-api-requests/user-signin/README.md:139]:
```suggestion
  // Dialogue 2: And use it after the h1 tag
```

File: making-api-requests/user-signin/README.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+Then, inside the signin `folder` I'll add a new file called `index.tsx` with the following content:
+```tsx
+import React, { useState } from 'react';
+
+// Dialogue 1: Let's define the Signin component
+const Signin: React.FC<> = () => {
+  return (
+    <div>
+      { /* Dialogue 2: with a basic h1 tag, Sign in */}
```
[the remark this answers is not in the document]
Comment (avishekjana) [reply making-api-requests/user-signin/README.md:17]: I've added dialogues for video recording purpose. When you are uploading a lesson as text lesson, you can skip these parts.

File: making-api-requests/user-signin/README.md (hunk 1 above)
Comment (avishekjana) [reply making-api-requests/user-signin/README.md:6]: Updated script and fixed it.

File: making-api-requests/user-signin/README.md (hunk 3 above)
Comment (avishekjana) [reply making-api-requests/user-signin/README.md:58]: Added URL

File: making-api-requests/user-signin/README.md (hunk 4 above)
Comment (avishekjana) [reply making-api-requests/user-signin/README.md]: Fixed.

**Pull request merged**